README.md

```diff
@@ -286,12 +316,9 @@ For more examples, I encourage you to check out the code for my demo app. It is
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v3.24 or above<sup>1</sup>
-* Ember CLI v3.24 or above
-* Node.js v12 or above
-* Modern browsers<sup>1</sup> (IE 11 won't be supported)
```

Comment:
[Ember v4 supports modern browsers only](https://emberjs.com/browser-support/). I'll go ahead with removing this line to simplify the `README`.